locus.html

```diff
@@ -12,18 +12,151 @@
+          <li><a href="#">Contact</a></li>
+          <li><a href="#">Location</a></li>
+          <li><a href="#">About us</a></li>
+        </ul>
+        <ul>
+          <li>
+            <a href="#">Sign Up</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <h1>Locus - your design</h1>
+      <section>
+        <h2>Get the inspiration of interior design here</h2>
+        <p>
+          architecture not only about engineering, it even lands to art and
+          aesthetics. With us, you will get a residentical design with an
+          extraordinary touch of art.
+        </p>
+        <button type="Let's go">Let's go</button>
+      </section>
+      <section>
+        <h2>Hot it works</h2>
+        <p>Spice up your space even simpler than you think</p>
+        <h3>
+          We provide design and build for commercial building, interior and
+          furniture to improve the better life.
+        </h3>
+        <button type="see video">see video</button>
+      </section>
+      <section>
+        <h2>Our mission is to deliver aesthetic visual for your home</h2>
+        <p>
+          By applying bacis psychology principles, we are going to explain how
+          perception works and how we can use this understanding to maximise the
+          space that we occupy in our homes.
+        </p>
+      </section>
+
+      <section>
+        <h2>Out project</h2>
+        <p>Introducing our first official project</p>
+        <ul>
+          <li>
+            <h3>Project Livingroom</h3>
+            <p>
+              A simple guide to create the perfect livingroom for. Night Watc
+              works beautifully in a wide gamut of arrangements.
+            </p>
+          </li>
+          <li>
+            <h3>Project workspace</h3>
+            <p>
+              A simple guide to create the perfect livingro- om for . Night Watc
+              works beautifully in a wi- de gamut of arrangements.
+            </p>
+          </li>
+          <li>
+            <h3>Library</h3>
+            <p>
+              A simple guide to create the perfect livingro om for. Night Watc
+              works beautifully in a wide gamut of arrangements.
+            </p>
+          </li>
+        </ul>
+      </section>
+      <section>
+        <p><b>105</b> project done</p>
+        <p><b>12</b> years of experience</p>
+        <p><b>10</b> big company partnership</p>
+        <span>
+          "Not able to tell you how happy I am with interior design. Interior
+          design has completely surpassed our expectations. Interior design
+          saved my business. Interior design is worth much more than I paid." -
+          Roscoe
+        </span>
+      </section>
+      <section>
+        <h2>Contact Us</h2>
+        <p>Have any questions? Want help before getting started?</p>
+      </section>
+      <form action="#" method="GET">
+        <p>
+          <label for="name"></label>
```

Comment:
все label у тебя пустые, добавь к ним что-нибудь, например
```suggestion
          <label for="name">Full name</label>
```